### PR TITLE
version bump and changelog update for 1.1.15

### PR DESCRIPTION
**Pull request by aaronchung-bitquill**
### Summary

version bump and changelog update for 1.1.15

### Description

version bump and changelog update for 1.1.15

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [1.1.14] - 2024-03-04
```
Comment (crystall-bitquill) [inline CHANGELOG.md:6]:
```suggestion
## [1.1.15] - 2024-06-05
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
...
+### Fixed
+* Exception thrown when using IAM authentication and CN RDS URL ([Issue](https://github.com/awslabs/aws-mysql-jdbc/issues/579)).
```
Comment (crystall-bitquill) [inline CHANGELOG.md:8]:
```suggestion
* Exception thrown when using IAM authentication and CN RDS URL ([Issue #579](https://github.com/awslabs/aws-mysql-jdbc/issues/579)).
```

File: README.md
Comment (crystall-bitquill) [inline README.md:1]: I think the snapshot version needs to be updated as well

File: CHANGELOG.md (hunk 1 above)
Comment (aaronchung-bitquill) [reply CHANGELOG.md]: TY fixed!

File: CHANGELOG.md (hunk 2 above)
Comment (aaronchung-bitquill) [reply CHANGELOG.md]: TY fixed!

File: README.md
Comment (aaronchung-bitquill) [reply README.md:1]: TY fixed!

**Review by crystall-bitquill — APPROVED** [review]

**Pull request merged**